build.gradle

```diff
@@ -236,30 +233,17 @@ configure(rootProject) { project ->
   }
 
 
-  if (project.hasProperty('platformVersion')) {
-	apply plugin: 'spring-io'
-
-	repositories {
-	  maven { url 'https://repo.spring.io/libs-snapshot' }
-	}
-
-	dependencyManagement {
-	  springIoTestRuntime {
-		imports {
-		  mavenBom "io.spring.platform:platform-bom:$platformVersion"
-		}
-	  }
-	}
-  }
-
-
   jar {
 	manifest {
-	  instruction 'Import-Package', bundleImportPackages.join(',')
 	  attributes("Created-By": "${System.getProperty("java.version")} (${System.getProperty("java.specification.vendor")})",
 			  "Implementation-Title": project.name,
 			  "Implementation-Version": project.version)
 	}
+	bnd(
+			"-exportcontents": "!*internal*,*;-noimport:=true",
+			"-nouses": "true",
```

Comment:
Why "-nouses=true"?

Reply:
copied over from the PR in addons that introduced BND, I have removed it in latest commit